fdroid/config.yml

```diff
@@ -0,0 +1,18 @@
+---
+repo_url: https://mobileapp.bitwarden.com/fdroid/repo/
+servergitmirrors: https://raw.githubusercontent.com/bitwarden/f-droid/main/fdroid/repo
+repo_name: Bitwarden F-Droid
+repo_description: >-
+  This is a repository of Bitwarden apps to be used with F-Droid. Applications
+  in this repository are official binaries built by Bitwarden.
+
+archive_url: https://mobileapp.bitwarden.com/fdroid/archive/
+archive_name: Bitwarden F-droid Archive
+archive_description: >-
+  This is a repository of archived Bitwarden apps that are no longer 
+  officially supported.
+
+archive_older: 10
+
+repo_keyalias: bitwarden-Virtual-Machine
+keydname: CN=bitwarden-Virtual-Machine, OU=F-Droid
```

Comment:
:pick:
```suggestion
keydname: CN=bitwarden-Virtual-Machine, OU=F-Droid

```

Reply:
🤔 That's how vscode saves the file but it's removed here?